libcontainer/cgroups/systemd/v2.go

```diff
@@ -263,11 +274,23 @@ func (m *unifiedManager) GetStats() (*cgroups.Stats, error) {
 }
 
 func (m *unifiedManager) Set(container *configs.Config) error {
-	fsMgr, err := m.fsManager()
-	if err != nil {
+	var (
+		c        = m.cgroups
+		unitName = getUnitName(c)
+	)
+
+	if properties, err := genResourcesProperties(c); err != nil {
 		return err
+	} else {
+		dbusConnection, err := getDbusConnection()
+		if err != nil {
+			return err
+		}
+		if err := dbusConnection.SetUnitProperties(unitName, true, properties...); err != nil {
+			return err
+		}
+		return nil
 	}
-	return fsMgr.Set(container)
```

Comment:
some resources are not controlled by systemd so we still need to call this I'm afraid.

Reply:
and this is why CI is failing (systemd does not control cpuset)

Reply:
Force pushed now.
I don't know whether we should change to only write `cpuset` cgroup files or not?

Reply:
I think the best is to keep it like it is now (with writing all of them), and then we can look at it in another PR/issue. This also applies to other resources not set via systemd like hugetlb. 

More info about it here: https://systemd.io/CGROUP_DELEGATION/